controllers/topics.controllers.js

```diff
@@ -0,0 +1,7 @@
+const { selectTopics } = require("../models/topics.models");
+
+exports.getTopics = (req, res) => {
+  selectTopics().then((topics) => {
+      res.status(200).send({ "topics": topics.rows });
+  });
```

Comment:
Rather than sending topics.rows here, we should be sending the rows from the model. The controller would then just call the model and send the response. :)